exporter/stats/stackdriver/stackdriver.go

```diff
@@ -309,3 +312,24 @@ func namespacedViewName(v string, escaped bool) string {
 	}
 	return path.Join("custom.googleapis.com", p)
 }
+
+func newLabels(tags []tag.Tag) map[string]string {
+	labels := make(map[string]string)
+	for _, tag := range tags {
+		labels[tag.Key.Name()] = tag.Value
+	}
+	return labels
+}
+
+func newLabelDescriptors(keys []tag.Key) []*labelpb.LabelDescriptor {
+	labelDescriptors := make([]*labelpb.LabelDescriptor, len(keys))
```

Comment:
This is setting the slice size to len(keys), not its capacity.  So:
labelDescriptors := make([]*labelpb.LabelDescriptor, 0, len(keys))

Reply:
Updated in https://github.com/census-instrumentation/opencensus-go/pull/167